cli/compose/convert/compose.go

```diff
@@ -81,3 +84,27 @@ func Networks(namespace Namespace, networks networkMap, servicesNetworks map[str
 
 	return result, externalNetworks
 }
+
+// Secrets converts secrets from the Compose type to the engine API type
+func Secrets(namespace Namespace, secrets map[string]composetypes.SecretConfig) ([]swarm.SecretSpec, error) {
+	result := []swarm.SecretSpec{}
+	for name, secret := range secrets {
+		if secret.External.External {
+			continue
+		}
+
+		data, err := ioutil.ReadFile(secret.File)
```

Comment:
I know we are doing an `ioutil.ReadAll` in `secret create`, but this doesn't seem right.
IIRC, we do enforce a max size on the daemon, I think we should check the file size before just reading it all into memory here.